pkg/importer/resource_loadbalancer_importer.go

```diff
@@ -9,8 +9,9 @@ import (
 
 func ResourceLoadBalancerStateGetter(obj *loadbalancerv1.LoadBalancer) (*StateGetter, error) {
 	states := map[string]interface{}{
-		constants.FieldCommonNamespace: obj.Namespace,
-		constants.FieldCommonName:      obj.Name,
+		constants.FieldCommonNamespace:       obj.Namespace,
+		constants.FieldCommonName:            obj.Name,
+		constants.FieldLoadBalancerIPAddress: obj.Status.Address,
```

Comment:
I'm not sure it's a good idea to go without any checks here. I think `obj.Status` or `obj.Status.Address` may be `nil` and I'm not sure this would work in that case.